neural_compressor/evaluation/lm_eval/models/huggingface.py

```diff
@@ -511,7 +514,6 @@ def _create_model(
                 if model_kwargs.get("load_in_4bit", None):
                     if model_kwargs.get("bnb_4bit_compute_dtype", None):
                         model_kwargs["bnb_4bit_compute_dtype"] = get_dtype(model_kwargs["bnb_4bit_compute_dtype"])
-            from neural_compressor.transformers import AutoModelForCausalLM
```

Comment:
why remove this line? if you remove this line, woq model generated by inc will load failed.

Reply:
Hi @changwangss  I think it's already import at the beginning
https://github.com/intel/neural-compressor/blob/b6b7d7c3c415d67976e054ab5ad5be6b5d5b460d/neural_compressor/evaluation/lm_eval/models/huggingface.py#L42

Reply:
ok, approve.